includes/collection/class-extra-fields.php

```diff
@@ -38,10 +39,11 @@ public static function get_actor_fields( $user_id ) {
 
 	public static function fields_to_attachments( $fields ) {
 		$attachments = array();
+		\add_filter( 'activitypub_urls_rel_me', '__return_true' );
```

Comment:
can we assume that a link in the extra fields is always a `me` link?

Reply:
Mastodon is doing it like that, so maybe it is the way to go! never mind, forget what I said :)